akka-stream/src/main/scala/akka/stream/impl/StreamLayout.scala

```diff
@@ -89,10 +186,10 @@ private[akka] object StreamLayout {
         attributes)
     }
 
-    def grow(that: Module): Module = grow(that, Keep.left)
+    def compose(that: Module): Module = compose(that, Keep.left)
 
-    def grow[A, B, C](that: Module, f: (A, B) ⇒ C): Module = {
-      if (Debug) validate()
+    def compose[A, B, C](that: Module, f: (A, B) ⇒ C): Module = {
```

Comment:
Thanks @viktorklang :) Once fixed give it some OK commit message and we'll merge.
The plan is to release 1.0 tomorrow.